_posts/2023-11-20-securing-wildfly-apps-okta-openshift.adoc

```diff
@@ -0,0 +1,266 @@
+---
+layout: post
+title: 'Securing an WildFly Applications Using Okta on OpenShift'
+date: 2023-11-20
+tags: oidc OpenShift auth0
+synopsis: A step by step guide on how to secure a WildFly application using Okta's OpenID Provider and deploy it to OpenShift. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly applications can be secured using OpenID Connect and deployed to OpenShift. By using OIDC to secure applications, you delegate authentication to OIDC providers. The `elytron-oidc-client` subsystem can be used to secure an application deployed to WildFly using any OpenID Provider. https://wildfly-security.github.io/wildfly-elytron/blog/securing-wildfly-apps-oidc-openshift/[A previous blog post] demonstrates how to secure an application using Keycloak, however, this guide demonstrates how to secure an example application deployed to WildFly on OpenShift using https://www.okta.com/[Okta] as the OpenID Provider. 
```

Comment:
For the guides, we've been adding links to relevant resources in the last section. So we could simplify this intro here to make it more similar to this guide:

https://wildfly-security.github.io/wildfly-elytron/blog/securing-wildfly-apps-auth0-openshift/

And then reference any links in the resources section.

Reply:
fixed!